netlab/L5/tls_protocol_layer.hpp

```diff
@@ -529,6 +531,7 @@ namespace netlab {
 						Extension extension(it, it + sizeof(Extension));
 						this->handshake.body.clientHello.extensions_union.extensions.push_back(extension);
 						it += sizeof(Extension);
+						i+= sizeof(Extension);
```

Comment:
Why do you need the second increment?

Reply:
no, in the for loop you have  (uint16_t i = 0; i < extensions_length; i++) 
but extensions_length is the total length in bytes and not the number ot extentions